.changesets/exp_qp_cache_prewarm_on_reload.md

```diff
@@ -1,12 +1,16 @@
 ### Allow disabling persisted-queries-based query plan cache prewarm on schema reload
 
-In Router v1.31.0, we started including operations from persisted query lists when Router pre-warms the query plan cache when loading a new schema.
+The router supports the new `persisted_queries.experimental_prewarm_query_plan_cache.on_reload` configuration option. It toggles whether a query plan cache that's prewarmed upon loading a new schema includes operations from persisted query lists. Its default is `true`. Setting it `false` precludes operations from persisted query lists from being added to the prewarmed query plan cache.
```

Comment:
@shorgi Should that say "Setting it to `false`"?

Also, could this be misread as stating that operations in PQLs are actively prevented from being prewarmed even if they show up via another mechanism of prewarming (ie the most recently used operations)? Probably not and this is OK.

Otherwise, edits look great!

Reply:
Good catch. I fixed this in the release prep step.

Reply:
I'm not sure I see that change in the merged PR, but nbd.